hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1130,22 +1128,6 @@ public void update(HoodieRollbackMetadata rollbackMetadata, String instantTime)
     }
   }
 
-  protected void validateRollback(
-      String commitToRollbackInstantTime,
-      HoodieInstant compactionInstant,
```

Comment:
We can do this now becase the MOR rollback is changed to base on file deletion similiar with the COW table.

Reply:
The rollback check needs to be kept because metadata such as RLI relies on the rollback of MDT commit, we can not do that if that commit has been compacted.